modules/nf-core/opt/flip/main.nf

```diff
@@ -0,0 +1,55 @@
+process OPT_FLIP {
+    tag "$meta.id"
+    label 'process_high'
+
+    container "khersameesh24/opt:v0.0.1"
+
+    input:
+    tuple val(meta), path(probes_fasta)
+    tuple val(meta2), path(ref_annot_gff), path(ref_annot_fa)
```

Comment:
maybe better to call them meta_probe and meta_ref to make clear also in the meta.yml explanation for what their are standing. For meta2, how do you envision sb. would use the module and populate meta2 information?